lmp/bb-build.sh

```diff
@@ -26,13 +26,18 @@ if [ "$BUILD_SDK" == "1" ] && [ "${DISTRO}" != "lmp-mfgtool" ]; then
 fi
 bitbake -D ${BITBAKE_EXTRA_ARGS} ${IMAGE}
 
-# write a summary of the buildstats to the terminal
-BUILDSTATS_SUMMARY="../layers/openembedded-core/scripts/buildstats-summary"
-if [ -f $BUILDSTATS_SUMMARY ]; then
-    BUILDSTATS_PATH="$(bitbake-getvar --value TMPDIR | tail -n 1)/buildstats"
-    if [ -d $BUILDSTATS_PATH ]; then
-        # get the most recent folder
-        BUILDSTATS_PATH="$(ls -td -- $BUILDSTATS_PATH/*/ | head -n 1)"
+# Check if the buildstats was enabled
+BUILDSTATS_PATH="$(bitbake-getvar --value TMPDIR | tail -n 1)/buildstats"
+if [ -d $BUILDSTATS_PATH ]; then
+    # get the most recent folder
+    BUILDSTATS_PATH="$(ls -td -- $BUILDSTATS_PATH/*/ | head -n 1)"
+    # producing bootchart.svg
+    run xvfb-run ../layers/openembedded-core/scripts/pybootchartgui/pybootchartgui.py \
```

Comment:
Can you add a check if xvfb is available and skip if not? Then this same logic would work when running ci-scripts on older lmp-sdks.

Reply:
good cacth! fixed